doc/data/messages/u/useless-import-alias/details.rst

```diff
@@ -0,0 +1,8 @@
+Known issue
+-----------
+
+If you prefer to use `from ... import ... as ...` because using
+`__all__` in `__init__.py` give error from other linter like `mypy`.
+Use `pylint` Message Control `pylint: disable=useless-import-alias`
+before any imports.
```

Comment:
```suggestion
If you prefer to use "from-as" to explicitly reexport in API (`from fruit import orange as orange`)
instead of using `__all__` this message will be a false positive.

If that's the case use `pylint: disable=useless-import-alias` before your imports in your API files.
```